taskcluster/translations_taskgraph/transforms/training_continuation.py

```diff
@@ -92,3 +92,24 @@ def add_pretrained_model_mounts(config, jobs):
                 }
 
         yield job
+
+
+evaluate = TransformSequence()
```

Comment:
nit: I feel like this is more of a "filter" than an evaluation.. but I'm very out of touch with nomenclature here, so it probably makes sense in context.

Reply:
Ah; in this case `evaluate` is a stage of the pipeline. What I intended by this is that this the transform(s) needed for training continuation support for the `evaluate` stage. (I can see that this is not obvious! I'll try to think of something better.)